### PR TITLE
Improve package's layout on mobile

### PR DESCRIPTION
**Pull request by Ayc0**
## TL;DR

Before | After
-|-
image | image

Continuation of https://github.com/yarnpkg/berry/pull/874

### Title

- Decrease size and margins

Before | After
-|-
image| image

### Layout

- use display flex instead of float
- use flex wrap and flex direction to put the description of the package 1st on the page on mobile
- remove margins around `<aside>` and `<main>` on mobile
- remove `<aside>`'s border left on mobile

### Display all button

- fix position of icon
- fix rotation of icon

Before | After
-|-
image | image

### Out of scope but I did it anyway (I did it because it's part of the package page)

I use [Kizu](https://twitter.com/kizmarh)'s [trick with object](https://www.kizu.ru/nested-links/) to allow nested copy button within link so now the entire `yarn` box is clickable and not only the text

image

### CODE REVIEW

File: packages/gatsby/src/components/details/Links.js [hunk 1]
```diff
@@ -42,10 +42,7 @@ const LinkBox = styled.span`
 
 export const Link = ({site, url, display, tag = 'a'}) => {
   const LinkElement = styled(tag)`
-    display: block;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
```
Comment (Ayc0) [inline packages/gatsby/src/components/details/Links.js]: those styles were duplicated below

File: packages/gatsby/src/images/search/ico-npm.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<svg width="34" height="13" viewBox="0 0 34 13" xmlns="http://www.w3.org/2000/svg">
+<svg width="34" height="17" viewBox="0 0 34 9" xmlns="http://www.w3.org/2000/svg">
```
Comment (Ayc0) [inline packages/gatsby/src/images/search/ico-npm.svg]: I noticed that the text wasn't centered so it caused alignment issue with `display: flex; align-items: center;`
